packages/ui/src/domains/LandingPageBuilder/LandingPageBuilderV2.tsx

```
@@ -0,0 +1,1098 @@
+					</Card>
+				);
+
+			case "form":
+				return (
+					<form className="space-y-4">
+						{component.content.fields.map((field, index) => (
+							<div key={index}>
+								<label className="block mb-1 font-medium text-gray-700">
+									{field.label}
+								</label>
+								{field.type === "textarea" ? (
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)